mmocr/models/textdet/heads/db_head.py

```diff
@@ -103,3 +84,101 @@ def _init_thr(self,
             nn.BatchNorm2d(inner_channels // 4), nn.ReLU(inplace=True),
             nn.ConvTranspose2d(inner_channels // 4, 1, 2, 2), nn.Sigmoid())
         return seq
+
+    def forward(self,
+                img: Tensor,
+                data_samples: Optional[List[TextDetDataSample]] = None,
+                mode: str = 'loss') -> Tuple[Tensor, Tensor, Tensor]:
```

Comment:
```suggestion
                mode: str = 'predict') -> Tuple[Tensor, Tensor, Tensor]:
```
better for inferencing